### PR TITLE
Adding change_0_to_5 code using python

### PR DESCRIPTION
**Pull request by SaiSakethAluru**

### CODE REVIEW

**Comment by SaiSakethAluru** [comment]
PR #623 done using python3

File: code/mathematical-algorithms/count_trailing_zeroes/Count_trailing_zeroes_factorial.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include <iostream>
```
Comment (AdiChat) [inline code/mathematical-algorithms/count_trailing_zeroes/Count_trailing_zeroes_factorial.cpp]: Kindly remove this code as it already exists 👍

File: code/mathematical-algorithms/count_trailing_zeroes/count_trailing_zeros_factorial.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+def findTrailingZeroes(n):
```
Comment (AdiChat) [inline code/mathematical-algorithms/count_trailing_zeroes/count_trailing_zeros_factorial.py]: This exists in previous pull request 👍

**Comment by AdiChat** [comment]
Kindly make a above changes to get this merged 👍

**Comment by SaiSakethAluru** [comment]
I'm really sorry, I'm new to git and github. So I probably made the same commits in different pull requests. Can you please specify how I can correct this? Thanks in advance

**Comment by SaiSakethAluru** [comment]
Removed the files. Please review and merge.

**Comment by AdiChat** [comment]
No problem 😄

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**